benchmark.sh

```diff
@@ -12,17 +12,23 @@ echo "Warming up Apache..."
 ab -c 100 -n 1000 -T 'application/json' http://$HOST:35357/ > /dev/null 2>&1
 
 echo "Benchmarking token creation..."
-ab -r -c 1 -n 200 -p auth.json -T 'application/json' http://$HOST:35357/v3/auth/tokens > results/create_token
+ab -r -e -c 1 -n 200 -p auth.json -T 'application/json' http://$HOST:35357/v3/auth/tokens > results/create_token
```

Comment:
hmm can't remember the specifics for adding the -e, but I guess the lines with the grep are enough. Will resubmit this PR.